plugins/nvme/fu-nvme-device.c

```diff
@@ -267,6 +267,33 @@ fu_nvme_device_dump(const gchar *title, const guint8 *buf, gsize sz)
 	g_print("\n");
 }
 
+/*
+ * Returns:
+ * %TRUE: device is in PCI subsystem
+ * %FALSE: device is, probably, NVMe-over-Fabrics
+ */
+static gboolean
+fu_nvme_device_is_pci(FuDevice *device, GError **error)
+{
+	g_autoptr(GUdevDevice) device_tmp = NULL;
+	GUdevDevice *gdev;
+
+	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
```

Comment:
is this strictly needed?  _Sometimes_ we do like to send `NULL` for the error argument to functions if we really don't care about the details of the outcome.

Reply:
In general we don't do these error checks for static functions. I'd say just remove the line then LGTM.

Reply:
Not strictly needed. While testing my patches, I was getting a segfault on error being dereferenced and I was checking whether it was me not setting error or if it was being derefereced without NULL checks. Turns out it was both. I should send a new patch for the second case though.
For now, I'll remove this line so we can go on.
Thanks.